views/spaces/index.erb

```diff
@@ -9,7 +9,7 @@
       <ul>   
       <%@spaces.each do|space|%>
         <li>
-        <%=space.name%>
+        <%=space.name%> - <%=space.description%>
```

Comment:
This is great for passing the test :1st_place_medal: . I guess later on when we might come to look at presentation this would change.

Reply:
Absolutely. We didn't consider presentation at all at this stage.